.travis.yml

```diff
@@ -10,10 +10,14 @@ branches:
 git:
   depth: 1
 
+jobs:
+  include:
+    - env: CMD="make ci e2e status=keep lighthouse=true"
+    - env: CMD="make ci e2e status=keep lighthouse=true globalnet=true"
+
 services:
   - docker
-script:
```

Comment:
Ah, you need something here or it just runs the default Make target and doesn't run e2e or use your custom CMDs at all.

```
  - set -o pipefail;
    $CMD 2>&1 | ts '[%H:%M:%.S]' -s
```

Reply:
Thanks. Saw this in submariner repo's travis.yml but wasn't sure if this indeed was the issue.